Add side-nav and profile component

tapeshmathur opened the pull request:
check it.

tapeshmathur committed:
second2

ambarvm commented on package.json:
```diff
@@ -22,6 +22,7 @@
 		"@angular/platform-browser": "~9.0.0",
 		"@angular/platform-browser-dynamic": "~9.0.0",
 		"@angular/router": "~9.0.0",
+		"a": "^2.1.2",
```
Please remove this line.

ambarvm commented on src/app/profile/profile.component.html:
```diff
@@ -0,0 +1,107 @@
+      <button (click)="showFiller = !showFiller" mat-raised-button>
+        Toggle extra text
+      </button> -->
+    </mat-drawer>
```
The side drawer should be in a separate component.

ambarvm commented on src/app/profile/profile.component.ts:
```diff
@@ -0,0 +1,19 @@
+export class ProfileComponent implements OnInit {
+  @Input() name;
+  @Input() desc;
+  @Input() dname;
```
I would suggest using more descriptive names like `description` and `displayName`. Also, it is considered a good practice to mention the type of the variable like `@Input() name: string;`.

ambarvm reviewed (changes requested):
Please have a look at the comments I have made and make changes accordingly. I have tried to incorporate all the best practices I know so far.

tapeshmathur committed:
changes

ambarvm committed:
🐛Bug fix and 🧹code cleaning
- refactor code so that both drawer and profile work
- remove unused typescript and css
- reformat existing code

ambarvm commented on src/app/profile/profile.component.html:
```diff
@@ -0,0 +1,41 @@
+				</mat-card-header>
+				<img
+					mat-card-image
+					src="https://media.npr.org/assets/img/2017/10/27/tra2016_comp_v005_008088.1111_wide-bc335cede10d086ba948db2d66f0093d36f0cc1e-s800-c85.jpg"
```
In profile component, we need **imageURL** as an **input** instead of a hard coded URL.

ambarvm reviewed (changes requested):
@tapeshmathur Please make changes as mentioned in review. Make sure to `git pull` before working as I have pushed some changes to your branch.

ambarvm committed:
update profile component
- use imageURL as input
- remove repetetive info
- remove unused image

ambarvm committed:
arrange imports in app module

ambarvm committed:
update fomatting
